gateway/src/apicast/configuration_loader/data_url.lua

```diff
@@ -0,0 +1,58 @@
+local MimeType = require('resty.mime')
```

Comment:
I think we need a module description explaining its purpose.

Reply:
Fixed.